### PR TITLE
Add url attribute to JSON object and updated google_rating to a string

### PR DESCRIPTION
**Pull request by holmesm8**
@caachz @agallant121 @TravisBorgsmiller 

This is updating our Sinatra API to return the JSON with a url so that we can include a link to that restaurant's website

### CODE REVIEW

File: app/facade/restaurant_facade.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class RestaurantFacade
```
Comment (TravisBorgsmiller) [inline app/facade/restaurant_facade.rb:1]: good looking fascade!

File: app/services/google_services.rb [hunk 2]
```diff
@@ -7,6 +7,6 @@ def initialize(phone_number)
     api = Faraday.new("https://maps.googleapis.com/maps/api/place/findplacefromtext/json?input=%2B#{number}&inputtype=phonenumber&fields=rating&key=#{ENV["GOOGLE_API_KEY"]}")
     response = api.get
     google = JSON.parse(response.body, symbolize_names: true)
-    @google_rating = google[:candidates][0][:rating]
+    @google_rating = google[:candidates][0][:rating].to_s
   end
```
Comment (TravisBorgsmiller) [inline app/services/google_services.rb:11]: Glad we got this changed

File: app/poros/restaurant.rb [hunk 3]
```diff
@@ -23,10 +24,11 @@ def initialize(restaurant)
     @longitude = restaurant[:coordinates][:longitude].to_s
     @location = restaurant[:location]
     @categories = restaurant[:categories]
+    @url = restaurant[:url]
```
Comment (agallant121) [inline app/poros/restaurant.rb:27]: Added website after it was shaken so users can get all the information they want from the restaurant's website directly

File: app/poros/restaurant.rb [hunk 4]
```diff
@@ -23,10 +24,11 @@ def initialize(restaurant)
...
   end
 
   def average_rating(google_rating)
     return @yelp_rating if google_rating == nil
-    (@yelp_rating.to_f + google_rating) / 2
+    (@yelp_rating.to_f + google_rating.to_f) / 2
```
Comment (agallant121) [inline app/poros/restaurant.rb:32]: Make sure ratings are returned as floats.

**Review by agallant121 — APPROVED** [review]

**Pull request merged**